tools/run_tests/helper_scripts/build_cxx.sh

```diff
@@ -24,15 +24,15 @@ INSTALL_PATH="$(pwd)/cmake/install"
 cd third_party/abseil-cpp
 mkdir build
 cd build
-cmake -DCMAKE_CXX_STANDARD=14 -DABSL_BUILD_TESTING=OFF -DCMAKE_BUILD_TYPE="${MSBUILD_CONFIG}" -DCMAKE_INSTALL_PREFIX="${INSTALL_PATH}" "$@" ..
+cmake -DABSL_BUILD_TESTING=OFF -DCMAKE_BUILD_TYPE="${MSBUILD_CONFIG}" -DCMAKE_INSTALL_PREFIX="${INSTALL_PATH}" "$@" ..
 make -j"${GRPC_RUN_TESTS_JOBS}" install
 
 # Install opentelemetry-cpp since we only support "package" mode for opentelemetry at present.
 cd ../../..
 cd third_party/opentelemetry-cpp
 mkdir build
 cd build
-cmake -DCMAKE_CXX_STANDARD=14 -DWITH_ABSEIL=ON -DBUILD_TESTING=OFF -DWITH_BENCHMARK=OFF -DCMAKE_BUILD_TYPE="${MSBUILD_CONFIG}" -DCMAKE_INSTALL_PREFIX="${INSTALL_PATH}" "$@" ..
+cmake -DWITH_ABSEIL=ON -DBUILD_TESTING=OFF -DWITH_BENCHMARK=OFF -DCMAKE_BUILD_TYPE="${MSBUILD_CONFIG}" -DCMAKE_INSTALL_PREFIX="${INSTALL_PATH}" "$@" ..
```

Comment:
as discussed we might need to make `-DCMAKE_CXX_STANDARD=<version>` a required argument set by the caller

Reply:
It turned out that we need this only for Mac so I'll add this only to Mac.